### DISCUSSION TITLE
Extend parameters from a template

### DISCUSSION THREAD

**Discussion by DanielHRio** [Azure/ResourceModules#2876] [Ideas]
Hello Team,

I can see a lot of parameters in .azuredevops/modulePipelines are duplicate. Would it be helpful if the parameters can be set in a seperated template? I can see [this feature request](https://developercommunity.visualstudio.com/t/Extend-parameters-from-a-template/1202473?from=email&viewtype=all&space=21&entry=suggestion+Visual+Studio+FeedbackVisual+Studio+Feedbackdevelopercommunity.visualstudio.com&q=parameters&ftype=idea) in VS Developer Community, but the number of votes is low. Would it be helpful if everyone can pump it up?

**Comment by eriqua** [comment]
Hi @DanielHRio , thanks for opening this discussion and sorry for the late reply. Discussions are currently not monitored promptly and are planned to be disabled in favor of the new issue template [Questions/Feedback].

I'm converting this discussion to an issue in order for the team to be able to triage it.

Please refer to issue #3142  going forward.
